dev/readthedocs.md

```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
+   python manage.py ingest jhu --years 2021 --terms fall
 
 2. How do I then load those courses into my database?
+   python manage.py digest jhu --years 2021 --terms fall
 
 3. How do I get a terminal running in my docker container?
+   docker-compose up
```

Comment:
How can you get a terminal in the container after you run `docker-compose up`?

Reply:
We can create a new terminal and run commands in that separate window. Hope I am understanding the question right.

Reply:
Not sure either, so I'll explain:

Docker pretty much creates its own environment (the "container") for code to be run. So you have your computer, and your computer has a docker container, and its inside the container that the local server is run (not on your actual computer). By creating a new terminal, you would end up with a terminal at the $HOME directory of your computer, not inside the docker container. The question is then, how do you create a new terminal linked to the docker container, and not your computer?

Reply:
Thanks for the clarification. We can use docker-compose run web /bin/bash to get a terminal in the container and use docker-compose run <container-name> <command> to run commands. Is this correct?

Reply:
or is it docker exec -it <container name> /bin/bash ?

Reply:
I know there *is* a way to do it with the command line, but I don't know it. The way I intended for you to do it is through the VS Code extension. See if you can find where I mentioned it in the docs